M2/Macaulay2/packages/AbstractSimplicialComplexes.m2

```diff
@@ -400,7 +396,7 @@ inducedSimplicialChainComplexMap(AbstractSimplicialComplex,AbstractSimplicialCom
 (
     h := simplicialChainComplex H;
     l := simplicialChainComplex L;
-    if areEqual(abstractSimplicialComplex {{}},H)==true then return map(l,h,zero)
+    if ((abstractSimplicialComplex {{}}) == H)==true then return map(l,h,zero)
```

Comment:
It's redundant to use `if x == true then ...`.  You can drop the `== true` and just use `if x then ...`.